### ISSUE TITLE
iceborne monster data not complete

### ISSUE THREAD

**Issue by valch1992** [LartTyler/MHWDB-Docs#63]
Hello, either im experiencing some kind of error or more than half of the iceborne monsters are missing, for example: monsters/46 and 47 are not found but i can access monsters like rajang or viper tobi kadachi.

**Comment by DeveloperPaul123** [comment]
No, you are correct. Many of the monsters are missing. I'm working on adding the rest of the IB monsters. They should be added soon.

**Comment by valch1992** [comment]
thank you for the fast response. are you adding them manually?

**Comment by DeveloperPaul123** [comment]
No, I'm going to be pushing the data using a script.

**Comment by VinsGecko** [comment]
Any news @DeveloperPaul123 ?
